packages/geospatial/package.json

```diff
@@ -22,6 +22,8 @@
   },
   "devDependencies": {
     "@performant-software/webpack-config": "^1.0.0",
+    "@peripleo/maplibre": "^0.3.2",
```

Comment:
Should these be in `dependencies` instead of `devDependencies`?

Reply:
Done!